.vscode/extensions.json

```diff
@@ -1,7 +1,6 @@
 {
   "recommendations": [
     "dbaeumer.vscode-eslint",
-    "esbenp.prettier-vscode",
-    "hbenl.vscode-jasmine-test-adapter"
+    "esbenp.prettier-vscode"
```

Comment:
can we add Jest extension as recommended?

Reply:
Done 👍